app/controllers/workshops_controller.rb

```diff
@@ -0,0 +1,48 @@
+class WorkshopsController < ApplicationController
+  def index
+    @workshops = Workshop.all
+  end
+
+  def show
+    @workshop = Workshop.find(params[:id])
```

Comment:
Maybe you want to reuse this code. Place it in a private method and use an before action?